DB/index.js

```diff
@@ -14,3 +14,27 @@ const db = mongoose.connection;
 
 
 module.exports = db;
+
+
+const {Client} = require('pg');
+console.log('first');
+const {user} = require('./environment.js');
+const {password} = require('./environment.js');
+const {host} = require('./environment.js');
+const {port} = require('./environment.js');
+const {database} = require('./environment.js');
+console.log('second');
+

```

Comment:
good use of environment variables! maybe check out the node package "dotenv" at some point, if you haven't already